mmv1/products/compute/FirewallPolicyRule.yaml

```diff
@@ -107,6 +115,33 @@ properties:
           CIDR IP address range. Maximum number of destination CIDR IP ranges allowed is 5000.
         item_type:
           type: String
+      - name: 'srcNetworkScope'
+        type: Enum
+        description: |
+          Network scope of the traffic source.
+        min_version: beta
```

Comment:
Looks like the added fields are `beta` only, could you add `(beta)` to the entries in the release note?

Reply:
@hao-nan-li done!